MAUI/DataGrid/Column-types.md

```diff
@@ -0,0 +1,593 @@
+
+* **AspectFill**: Clips the image so that it fills the display area while preserving the aspect (no distortion).
+
+* **AspectFit**:  Letterboxes the image (if required) so that the entire image fits into the display area, with blank space added to the top/bottom or sides depending on whether the image is wide or tall.
+
+* **Fill**: Stretches the image to completely and exactly fill the display area. This may result in the image being distorted.
+
+* **Center**: Places the image to the center of the cell.
+
+To set `Aspect` to images loaded inside `DataGridImageColumn`, refer the below code snippet.
+
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<ContentPage.BindingContext>
+    <local:ViewModel />
+</ContentPage.BindingContext>
+
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                   AutoGenerateColumnsMode="None"
+                   ItemsSource="{Binding OrdersInfo}">
+    <syncfusion:SfDataGrid.Columns>
+        <sfGrid:DataGridImageColumn MappingName="DealerImage" Aspect="AspectFit"/>
+    </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+
+## DataGridTemplateColumn
+
+The DataGridTemplateColumn is derived from DataGridColumn. Hence, it inherits all the properties of DataGridColumn. It allows you to extend the functionality of DataGridColumn with own view by creating the [CellTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridTemplateColumn.html#Syncfusion_Maui_DataGrid_DataGridTemplateColumn_CellTemplate).
+
+The following table provides the list of properties in DataGridTemplateColumn:
+
+<table>
+<tr>
+<th>Property</th>
+<th>Type</th>
+<th>Description</th>
+<th>Default Value</th>
+</tr>
+<tr>
+<td>CellTemplate</td>
+<td>DataTemplate</td>
+<td>Gets or sets the template that is used to display the contents of the record cells.</td>
+<td>Null</td>
+</tr>
+</table>
+
+### Cell template
+
+Underlying records will be the BindingContext for the `CellTemplate`. The following code example shows templating of DataGridTemplateColumn:
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:DataGridTemplateColumn MappingName="CustomerID" HeaderText="Customer ID">
+    <syncfusion:DataGridTemplateColumn.CellTemplate>
+        <DataTemplate>
+            <StackLayout>
+                <Label Text="{Binding CustomerID}" TextColor="Blue" />
+            </StackLayout>
+        </DataTemplate>
+    </syncfusion:DataGridTemplateColumn.CellTemplate>
+</syncfusion:DataGridTemplateColumn>
+
+### CellTemplateSelector
+
+Load the underlying records through the `CellTemplate` to use CellTemplateSelector. The following code example shows templating of the DataGridTemplateColumn using the `CellTemplate` property:
```

Comment:
@NirmalKumarYuvaraj  - Replace the below content,

You can load any view to the cells through the `CellTemplate` by assigning the `TemplateSelector`.

Reply:
Replaced the content.